docs/_glossary/Delimiter.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Delimiter
-show-in: [ug, dg]
+show-in: [dg]
```

Comment:
"Delimiter" is used in the UG. Let's change this back.

```suggestion
show-in: [ug, dg]
```

Reply:
my bad was wanting to remove `dg`  instead of `ug`. Delimiter wasn't used in DG  (at least as of now)

Reply:
I'll remove it from DG for now, leave it in UG. Can add back into DG if you think it is relevant